Chinese_Remainder_Theorem/chinese_remainder_theorem.c

```diff
@@ -0,0 +1,71 @@
+#include <stdio.h>
+
+//function to find minimum number such that num%divisor[i] gives corresponding remainder[i] 
+void minresult( int divisor[], int rem[], int size)
+{
+    //each number starting from 1 is checked for minimum value
+    int res = 1, i = 0;
+    do
+    {  
+        if((res%divisor[i]) == rem[i])
+        {
+ // When value of i is equal to value of size i.e. all remainder matches with given rem[] array, then res is the minimum value
+            i++ ;
+        }
+        else
+        {    //if above condition gives false for any value of i, check for next number.
+            res++ ;
```

Comment:
Your code gives a brute force solution. It's not CRT.

Reply:
Sir this method is also used in code implemented using cpp so I thought this method is acceptable.

Reply:
@raghav-dalmia

Reply:
It will give the same output, but that's not CRT.

Reply:
Ok sir. Will try to implement CRT.

Reply:
Change your code. It's not acceptable for CRT. You can create a different issue for this solution(if it's not present).